src/StatusQ/Core/Theme/StatusDarkTheme.qml

```diff
@@ -110,6 +110,7 @@ ThemePalette {
 
     successColor1: getColor('green3')
     successColor2: getColor('green3', 0.2)
+    successColor3: getColor('green3', 0.3)
```

Comment:
This doesn't seem to exist in the design system: https://www.figma.com/file/MtAO3a7HnEH5xjCDVNilS7/%F0%9F%8E%A8-Design-System-%E2%8E%9C-Desktop?node-id=1%3A10

Has this color been discussed with design?

Reply:
So generally I'd say, let's just leave it as is. But as soon as we rely to much on `successColor3` in the future, this can become a hassle if there was indeed an officially introduced `successColor3`.

I guess it's fine to take this risk, as the color system hasn't been touched since we set it up originally, so it should be quite timeliness. Nonetheless, I wouldn't mind, if you double check with the design team if there's plans for such a color in our color system, and at least bring this to their attention.

Reply:
> Do you think in this specific Success case, border colours will be like background (so using successColor2)?

Actually I think we should do this instead. Turns out there is indeed designs of a success variant that do exactly that.